### PR TITLE
Introduce lodash deepcopy lib

### PR DESCRIPTION
**Pull request by gasiortomasz**

### CODE REVIEW

File: universal-login-commons/test/core/utils/deepCopy.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import {expect} from 'chai';
-import {deepCopy} from '../../../lib/core/utils/deepCopy';
+const deepCopy = require('lodash.clonedeep');
 
 describe('deepCopy', () => {
```
Comment (JustynaBroniszewska) [inline universal-login-commons/test/core/utils/deepCopy.ts]: Do we need to test the imported library?

File: universal-login-commons/lib/core/utils/deepMerge.ts [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import {deepCopy} from './deepCopy';
+const deepCopy = require('lodash.clonedeep');
```
Comment (marekkirejczyk) [inline universal-login-commons/lib/core/utils/deepMerge.ts]: import clonedeep from 'lodash.clonedeep';

File: universal-login-commons/lib/core/utils/securityCodes.ts [hunk 3]
```diff
@@ -4,6 +4,7 @@ import {ensure} from './errors';
 import {isProperAddress, reverseHexString} from './hexStrings';
 import {shuffle, array8bitTo16bit} from './arrays';
 import {Notification} from '../models/notifications';
+const deepCopy = require('lodash.clonedeep');
```
Comment (marekkirejczyk) [inline universal-login-commons/lib/core/utils/securityCodes.ts]: Don't use require, rename `deepCopy` to `clonedeep`, everywhere in the project. We don't code to reflect history.

File: universal-login-commons/test/core/utils/deepCopy.ts (hunk 1 above)
Comment (marekkirejczyk) [reply universal-login-commons/test/core/utils/deepCopy.ts]: We can now remove that file.

**Pull request merged**